zgw_consumers/admin.py

```diff
@@ -40,6 +41,20 @@ class CertificateAdmin(PrivateMediaMixin, admin.ModelAdmin):
 
     private_media_fields = ("public_certificate", "private_key")
 
+    def expiry_date(self, obj=None):
+        # alias model property to catch file not found errors
+        try:
+            return obj.expiry_date
+        except FileNotFoundError:
+            return _("file not found")
+
+    def is_valid_key_pair(self, obj=None):
+        # alias model method to catch file not found errors
+        try:
+            return obj.is_valid_key_pair()
+        except FileNotFoundError:
+            return _("file not found")
```

Comment:
I'd rather see this handled on the model methods/properties, since calling that code somewhere else than in the admin would still cause crashes.